di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -318,13 +333,30 @@ public AuthorizeHandler(
                             new Credential(
                                     credentialAttributesMap, gpgMap, userId, clientIdValue, exp);
 
+                    boolean F2F_SEND_VC_QUEUE =
+                            Objects.equals(
+                                    queryParamsMap.value(RequestParamConstants.F2F_SEND_VC_QUEUE),
+                                    "checked");
+                    if (F2F_SEND_VC_QUEUE) {
+                        String signedVcJwt =
+                                verifiableCredentialGenerator.generate(credential).serialize();
+                        HTTPRequest httpRequest =
+                                new HTTPRequest(
+                                        HTTPRequest.Method.POST, URI.create(F2F_STUB_QUEUE_URL));
+                        ObjectMapper objectMapper = new ObjectMapper();
+                        F2FEnqueueLambdaRequest enqueueLambdaRequest =
+                                new F2FEnqueueLambdaRequest(
+                                        F2F_STUB_QUEUE_NAME,
```

Comment:
I'd make the queue name (or environment) an input field on the stub page (maybe pre-seeded with either F2F_STUB_QUEUE_NAME or ${environment} so that we can use the build/prod stubs in dev accounts?

Reply:
Done 👍